.github/workflows/ios-testflight.yml

```
@@ -0,0 +1,392 @@
+
+          git config user.name "GitHub Actions"
+          git config user.email "actions@github.com"
+
+          git tag -a "$TAG_NAME" -m "TestFlight build $BUILD_VERSION"
+          git push origin "$TAG_NAME" || echo "Warning: Could not push tag"
+
+          echo "Tagged as $TAG_NAME"
+
+      - name: Clean up keychain
+        if: always()
+        run: security delete-keychain $RUNNER_TEMP/app-signing.keychain-db || true
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:26: Double quote to prevent globbing and word splitting

(shellcheck)